airbyte-commons-worker/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -109,6 +111,16 @@ public static WorkerDestinationConfig syncToWorkerDestinationConfig(final Standa
         .withState(sync.getState());
   }
 
+  public static Optional<AirbyteControlConnectorConfigMessage> getLastConfigControlMessage(final Map<Type, List<AirbyteMessage>> messagesByType) {
```

Comment:
nit: `last` could be ambiguous.  Perhaps `getMostRecentConfigControlMessage` could be clearer?

Reply:
👍🏻 done in [d94feb1](https://github.com/airbytehq/airbyte/pull/20894/commits/d94feb1d439f6aef4e876428965b1534cf5f3d65)